pkg/netpol/connlist/exposure_analysis.go

```diff
@@ -13,17 +13,24 @@ type exposureMaps struct {
 
 // peerXgressExposureData store exposure data of a peer on one direction ingress/egress
 type peerXgressExposureData struct {
-	isProtected  bool
+	// isProtected indicates whether the peer is protected by network-policies on the relevant direction or not
+	isProtected bool
+	// exposureInfo list of the exposure-connections data of the peer on the relevant direction.
+	// if isProtected is false (peer is not protected), this field will be empty
 	exposureInfo []*xgressExposure
 }
 
 // ----------------------------------------------------
 // xgressExposure implements XgressExposureData interface
 type xgressExposure struct {
+	// exposedToEntireCluster indicates if the peer is exposed to all namespaces in the cluster for the relevant direction
```

Comment:
if this field is true, then `namespaceLabels` and `podLabels` are empty?

Reply:
yes, updated comments